### ISSUE TITLE
virus

### ISSUE THREAD

**Issue by bowfr** [UnamSanctam/SilentCryptoMiner#1978]
windows says its a virus do i trust it or no

**Comment by UnamSanctam** [comment]
Do you mean the builder? If so then yes, it's open source, you can see its code here on GitHub.

**Comment by bowfr** [comment]
thanks and when it says done i cant find the file it made can you help me

**Comment by bowfr** [comment]
> Do you mean the builder? If so then yes, it's open source, you can see its code here on GitHub.


thanks and when it says done i cant find the file it made can you help me

**Comment by UnamSanctam** [comment]
Where did you choose to built it? Then you press "Build" a save dialog will appear for you to choose where to build it.

**Comment by bowfr** [comment]
> Where did you choose to built it? Then you press "Build" a save dialog will appear for you to choose where to build it.

yeah i put it to desktop the first time i didnt see it then i put it to downloads and i dont see it

**Comment by UnamSanctam** [comment]
It should be there, a UCompiler folder should also be there. Make sure that your antivirus isn't removing the UCompiler folder or some compiler program inside it.

**Comment by bowfr** [comment]
i see the ucomplier folder what should i do now

**Comment by UnamSanctam** [comment]
The .exe file that you built should be in the same folder where the UCompiler folder is. Can you take a screenshot? You can try adding an exclusion to your antivirus for the folder you're building the miner in.

**Comment by bowfr** [comment]
i go in the folder and it shows me this 
![Screenshot 2023-11-27 041333](https://github.com/UnamSanctam/SilentCryptoMiner/assets/152131232/071e5456-51fb-450c-9c49-267e04f491f8)

**Comment by UnamSanctam** [comment]
I mean outside that folder, where the UCompiler is located.

**Comment by bowfr** [comment]
alright

**Comment by bowfr** [comment]
i can only see the thing it built when i click on build again it shows me the file

**Comment by bowfr** [comment]
![Screenshot 2023-11-27 043619](https://github.com/UnamSanctam/SilentCryptoMiner/assets/152131232/22b81ba4-ce73-47ec-baf6-f1716733bc94)
which one do i send for them to download

**Comment by UnamSanctam** [comment]
You should send `fnrbx.exe`. The uninstaller will uninstall the miner and the checker will check if the miner is running on the current computer.

**Comment by bowfr** [comment]
so once i send the miner and they click it will it run on their computer
and how do i get people to actually click it

**Comment by UnamSanctam** [comment]
It depends on your strategy, though if someone downloads your file then most will also start it (as long as they were expecting to start it).

**Comment by bowfr** [comment]
and they have to download it and open it right and if they open it what will it open as

**Comment by UnamSanctam** [comment]
>what strategy should i use

There are uncountable different strategies of spreading, the most common way is to spread it through fake or real program cracks or game hacks, but developing your own strategy is the optimal way.

>and they have to download it and open it right and if they open it what will it open as

Nothing will appear since it's a silent miner, if you want some sort of program to appear then you can use a file binder or downloader to bind the miner together with some program or file.

**Comment by bowfr** [comment]
so what if i say i have Fortnite cheats and create a Fortnite cheat gui (which i dont know how to do) would that be a good idea

**Comment by UnamSanctam** [comment]
Sure or use a cheat GUI that already exist, or you can pretend that there's a cheat GUI and just send the program. Though if there is some real GUI then it would be less suspicious compared to if nothing happens when they start the program.

**Comment by bowfr** [comment]
what should i do after i find the download to the gui

**Comment by bowfr** [comment]
im confused

**Comment by bowfr** [comment]
> I sell spreading methods as well as cheat GUI applications, when a cheat is opened the miner will drop and start working. Add me on Discord - gota3 for details @bowfr

ok added

**Comment by gabjohn3** [comment]
> windows says its a virus do i trust it or no

Please leave this repo and don't use this tool bro :)
